code/game/gamemodes/cult/cult_items.dm

```diff
@@ -93,4 +122,193 @@
+	if(!iscultist(user))
+		user.unEquip(src, 1)
+		user.Weaken(5)
+		to_chat(user, "<span class='warning'>A powerful force shoves you away from [src]!</span>")
+		return
+	if(curselimit > 1)
+		to_chat(user, "<span class='notice'>We have exhausted our ability to curse the shuttle.</span>")
+		return
+	if(shuttle_master.emergency.mode == SHUTTLE_CALL)
+		var/cursetime = 1500
+		var/timer = shuttle_master.emergency.timeLeft(1) + cursetime
+		shuttle_master.emergency.setTimer(timer)
+		to_chat(user,"<span class='danger'>You shatter the orb! A dark essence spirals into the air, then disappears.</span>")
+		playsound(user.loc, "sound/effects/Glassbr1.ogg", 50, 1)
+		qdel(src)
+		sleep(20)
+		var/global/list/curses
+		if(!curses)
+			curses = list("A fuel technician just slit his own throat and begged for death. The shuttle will be delayed by two minutes.",
+			"The shuttle's navigation programming was replaced by a file containing two words, IT COMES. The shuttle will be delayed by two minutes.",
+			"The shuttle's custodian tore out his guts and began painting strange shapes on the floor. The shuttle will be delayed by two minutes.",
+			"A shuttle engineer began screaming 'DEATH IS NOT THE END' and ripped out wires until an arc flash seared off her flesh. The shuttle will be delayed by two minutes.",
+			"A shuttle inspector started laughing madly over the radio and then threw herself into an engine turbine. The shuttle will be delayed by two minutes.",
+			"The shuttle dispatcher was found dead with bloody symbols carved into their flesh. The shuttle will be delayed by two minutes.",
+			"Steve repeatedly touched a lightbulb until his hands fell off. The shuttle will be delayed by two minutes.")
+		var/message = pick(curses)
+		command_announcement.Announce("[message]", "System Failure", 'sound/misc/notice1.ogg')
+		curselimit++
+
+/obj/item/device/cult_shift
+	name = "veil shifter"
+	desc = "This relic teleports you forward a medium distance."
+	icon = 'icons/obj/cult.dmi'
+	icon_state ="shifter"
+	var/uses = 2
+
+/obj/item/device/cult_shift/examine(mob/user)
+	..()
+	if(uses)
+		to_chat(user, "<span class='cult'>It has [uses] uses remaining.</span>")
+	else
+		to_chat(user, "<span class='cult'>It seems drained.</span>")
+
+/obj/item/device/cult_shift/proc/handle_teleport_grab(turf/T, mob/user)
+	var/mob/living/carbon/C = user
+	if(C.pulling)
+		var/atom/movable/pulled = C.pulling
+		pulled.forceMove(T)
+		. = pulled
+
+/obj/item/device/cult_shift/attack_self(mob/user)
+	if(!uses || !iscarbon(user))
+		to_chat(user, "<span class='warning'>\The [src] is dull and unmoving in your hands.</span>")
+		return
+	if(!iscultist(user))
+		user.unEquip(src, 1)
+		step(src, pick(alldirs))
+		to_chat(user, "<span class='warning'>\The [src] flickers out of your hands, too eager to move!</span>")
+		return
+
+	var/mob/living/carbon/C = user
+	var/turf/mobloc = get_turf(C)
+	var/turf/destination = get_teleport_loc(mobloc,C,9,1,3,1,0,1)

```

Comment:
This doesn't exclude space tiles, so you might want to include something like how [bluespace fruit](https://github.com/ParadiseSS13/Paradise/blob/daec491d4eb459ebdef6a4f655e91b08e1dd822d/code/modules/hydroponics/seed.dm#L378) handles the destination selection instead of the get_teleport_loc proc (which should probably just have a parameter for excluding space turf destinations)